consensus/istanbul/backend/api.go

```diff
@@ -294,8 +321,12 @@ func (api *APIExtension) makeRPCOutput(b *types.Block, proposer common.Address,
 		rpcTransactions[i] = klaytnApi.RpcOutputReceipt(tx, hash, head.Number.Uint64(), uint64(i), receipts[i])
 	}
 
-	r["committee"] = committee
-	r["proposer"] = proposer
+	r["committee"] = cInfo.committee
+	r["proposer"] = cInfo.proposer
+	r["round"] = cInfo.round
+	//if cInfo.round > 0 {
```

Comment:
Can I ask the reason why you put this comment?

Reply:
Ops... it's mistake. I will remove it.